tests/test_cacerts_from_env.py

```diff
@@ -0,0 +1,41 @@
+import os
+import mock
+import httplib2
+
+CA_CERTS_BUILTIN = os.path.join(os.path.dirname(httplib2.__file__), "cacerts.txt")
+ENV_CERTS_FILE = "unittest_env_certs_file"
+CERTIFI_CERTS_FILE = "unittest_certifi_file"
+CUSTOM_CA_CERTS = "unittest_custom_ca_certs"
+
```

Comment:
Can you add a new test case, remove certiwifi from sys.modules and ensure that the module still works?

Reply:
Done.